pageserver/src/tui_event.rs

```diff
@@ -53,6 +53,8 @@ impl Events {
             let ignore_exit_key = ignore_exit_key.clone();
             thread::spawn(move || {
                 let stdin = io::stdin();
+
+                #[allow(clippy::manual_flatten)]
                 for evt in stdin.keys() {
```

Comment:
Don't have a preference either way, but why the suppression vs using `stdin.keys().flatten()`

Reply:
Oops, this was not the code I thought it was-- I thought clippy was complaining about something that Heikki talked me out of changing in an earlier pass at clippy cleanup.

I think what's needed here is better error handling-- it wouldn't make any sense to spin in a loop if, for example, stdin returns EOF because the user's ssh session died.